backend/domain/Room.js

```diff
@@ -74,20 +62,12 @@ class Room {
         return playersOfTheRole;
     }
 
-    getHost() {
-        return this.Host;
-    }
-
     addPlayer(player) {
         if (player !== null && this.players.length < this.maxPlayerCount) {
             this.players.push(player);
         }
     }
 
-    getRoundNumber() {
-        return this.roundNumber;
-    }
-
     incrementRoundNumber() {
         this.roundNumber++;
```

Comment:
I am pretty sure ESLint will convert `++` into `+= 1` but @MKTPR's hasn't been merged in, so might be worth just changing it here anyway

Reply:
Ok yup, I can change that. Personally I don't see an issue with ++, but for consistency purposes will change it.